doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -33,7 +33,25 @@ Samples and applications
 
 This section describes the changes related to samples and applications.
 
-|no_changes_yet_note|
+Serial LTE Modem (SLM)
+----------------------
+
+.. toggle::
+
+  * The handling of Release Assistance Indication (RAI) socket options has been updated in the ``#XSOCKETOPT`` command.
+    The individual RAI-related socket options have been consolidated into a single ``SO_RAI`` option.
```

Comment:
I think that this was documented already when the change was introduced?

Reply:
It was introduced in our integration branch here back in 2.6.0 https://github.com/lemrey/sdk-nrf/pull/127/files but by mistake it got lost in the release notes PR here https://github.com/nrfconnect/sdk-nrf/pull/14298

Reply:
Though the release notes do include this information: https://github.com/nrfconnect/sdk-nrf/pull/14298/files#diff-3f912a1116dc4be34d572163214169f5cc1467f65c3734da36b6078cd075d3d7R1123
Whichever way is fine.